vortex-file/src/strategy.rs

```diff
@@ -179,6 +179,8 @@ impl LayoutWriter for BtrBlocksCompressedWriter {
             }
         };
 
+        compressed_chunk.statistics().inherit(chunk.statistics());
```

Comment:
Inheriting stats is slow, but I agree we probably need to? I don't recall actively choosing to remove this